### DISCUSSION TITLE
Conflicting myflashlab ane with distriqt ANE

### DISCUSSION THREAD

**Discussion by balaedit** [airsdk/Adobe-Runtime-Support#2131] [General]
I have the project working with distriqt ANE and myflashlab ANE. But issue is the getting error in duplicate classes.

Which ANE of overrideAir.ane of myflashlab have same classes of distriqt ANE's.

@marchbold pls help to resolve the issue.

**Comment by marchbold** [comment]
Hi, Most likely it is `com.distriqt.Core` that is conflicting with the overrideAir extension.

Try using the [com.distriqt.Core.noair.ane](https://github.com/distriqt/ANE-Core/blob/master/lib/noair/com.distriqt.Core.noair.ane) variant instead. 

We have this version to resolve this exact issue.

Note: it still has the same extension id `com.distriqt.Core`

**Reply by balaedit** [reply]
Ok. Thanks. I will check
